src/network/networkpacket.h

```diff
@@ -40,7 +40,7 @@ class NetworkPacket
 	u32 getSize() const { return m_datasize; }
 	session_t getPeerId() const { return m_peer_id; }
 	u16 getCommand() { return m_command; }
-	const u32 getRemainingBytes() const { return m_datasize - m_read_offset; }
+	const u32 getRemainingBytes() { return m_datasize - m_read_offset; }
```

Comment:
I think you removed the wrong const here
```suggestion
	u32 getRemainingBytes() const { return m_datasize - m_read_offset; }
```

Reply:
This is what [LGTM](https://lgtm.com) suggested:

```
The `const` modifier has no effect on return types. The `const` modifying the return type can be removed.
```

https://lgtm.com/projects/g/minetest/minetest/latest/files/src/network/networkpacket.h?sort=name&dir=ASC&mode=heatmap

Reply:
This is exactly what @sfan5 proposed: remove const from u32, not from the method.